presto-docs/src/main/sphinx/release/release-0.214.rst

```diff
@@ -0,0 +1,63 @@
+=============
+Release 0.214
+=============
+
+General Changes
+---------------
+
+* Fix memory leak related to query tracking in coordinator that was introduced
+  in 0.213.
+* Send proper error response to client for certain types of errors during query
+  creation.
+* Improve error message when comparator provided to :func:`array_sort` function
+  doesn’t satisfy the required contract.
+* Improve query performance...??Lazily build hash table for ``MapBlock`` when
+  the hash table is used in ``SingleMapBlock``.
+* Improve planning more stable with regard to query predicates and joins.
+* Make distributed ``ORDER BY`` and aggregation unspill progress on early yield
+  signals.
+* Make top-N row number optimization work in some cases when columns are pruned.
+* Faster range reduction function in map hash table operations.
```

Comment:
Should this be in a form of a sentence?

Reply:
Also, this still is too much implementation detail. I’d just say “improve performance for lookup operations on map data types” (assuming that’s where it helps). We can link to the issue if needed for anyone that wants more details.

Reply:
The associated commit was 290f5be. The changes are in the same scope as 23de11f and improve the performance of building the internal hash table that represents map data types.

I'm keeping your suggestions:
> * Improve performance for lookup operations on map data types.

And using it for the two commits rather than providing two low-level entries.